api/lib/domain/events/handle-certification-rescoring.js

```diff
@@ -123,15 +123,15 @@ async function _handleV3Certification({
     configuration,
   });
 
-  const competencesForScoring = await scoringConfigurationRepository.listByLocale({ locale });
+  const v3CertificationScoring = await scoringConfigurationRepository.listByLocale({ locale });
```

Comment:
Je trouve que le v3 à la fin simplifie la lecture
```suggestion
  const certificationScoringV3 = await scoringConfigurationRepository.listByLocale({ locale });
```

Reply:
Commentaire pour la classe associé :D